### DISCUSSION TITLE
Not able to perform sql command

### DISCUSSION THREAD

**Discussion by mehulsatardekar** [supabase/supabase#7167] [Questions]
I am kind of new to supabase I am basically trying to perform pgsql command to describe table schema  
`\d table name` but it's giving me an error.

![pgs](https://user-images.githubusercontent.com/36101139/172051218-a9bbe794-18e4-4cad-8910-2369159173f2.PNG)

**Comment by GaryAustin1** [comment]
You are trying to run a psql terminal command.  The sql editor in the dashboard is for direct entry of sql.

https://www.postgresql.org/docs/current/app-psql.html About terminal command interface which is not in dashboard. 

https://linuxhint.com/describe-table-postgresql/ Shows ways to get table info with sql at bottom.
